caravel/assets/visualizations/filter_box.jsx

```diff
@@ -10,22 +10,80 @@ import '../stylesheets/react-select/select.less';
 
 import './filter_box.css';
 
+const TIME_CHOICES = [
```

Comment:
I'm putting the `constants.js` in viz folder to try to keep that contained somewhat